src/systemctl/systemctl.c

```diff
@@ -1167,3 +1168,4 @@ static int run(int argc, char *argv[]) {
 }
 
 DEFINE_MAIN_FUNCTION_WITH_POSITIVE_FAILURE(run);
+#endif
```

Comment:
hmm, instead of littering main programs with this ifdeffery, maybe a better aproach would be to add this conditionalization to main-func.h.

Specifically:

beef up main-func.h so that it honours a new define DISABLE_MAIN_FUNC or so: if set the generation of main() is simply suppresed, and DEFINE_MAIN_FUNCTION and friends become NOPs.

That way, the fuzzer could just set this, and systemctl.c itself would not have to be modified at all. (well except the exporting of parse_argv()). And it would work for similar programs the same way in the future